dev/tests/unit/phpunit.xml.dist

```diff
@@ -28,6 +28,7 @@
         </testsuite>
     </testsuites>
     <php>
+        <ini name="memory_limit" value="-1"/>
```

Comment:
Note: this change was done in order to remove the memory limit. Earlier, it was failing with fatal error:
![image](https://user-images.githubusercontent.com/1873745/128489461-d7f1978e-a2db-4a61-853c-3ef29e581d68.png)
After this change, the issue is gone:
![image](https://user-images.githubusercontent.com/1873745/128489522-689c1c76-b5ea-446c-a133-2e2c37cd757f.png)